code/00_setup-environment.R

```diff
@@ -93,7 +93,7 @@ postcode <- function(){
   clean_names() %>%
   
   select(pc7, ca2019, ca2019name, ca2018, hb2019, hb2019name,
-         hscp2019, hscp2019name, hscp2018, ur6_2016, 
+         hscp2019, hscp2019name, hscp2018, ur6_2016_name, 
```

Comment:
I noticed you also have 2-fold aggregated table in the first Excel workbook. So also need to extract ur2_2016_name?